### ISSUE TITLE
store bluesky and ophyd version in runengine metadata

### ISSUE THREAD

**Issue by prjemian** [bluesky/bluesky#1050]
Wouldn't it be nice if ...

## Expected Behavior
 ... the metadata informed us what versions of bluesky and ophyd were actually used for acquisition?

## Current Behavior
We can make this more common, doesn't do this now

## Possible Solution
I've got code

## Steps to Reproduce (for bugs)
enhancement

## Context
when we try to understand why and how things went wrong

## Your Environment
any

**Comment by stuartcampbell** [comment]
I'm always a fan of recording info like this... you never know when you need it.  PR ?

**Comment by prjemian** [comment]
yes, PR coming, just jotting down notes today with Tom

**Comment by prjemian** [comment]
looks like

```
RE.md['BLUESKY_VERSION'] = bluesky.__version__
RE.md['OPHYD_VERSION'] = ophyd.__version__
```

**Comment by stuartcampbell** [comment]
Should the ophyd version be at that level, or should it be at the descriptor level ?

**Comment by tacaswell** [comment]
Talking with @prjemian in person we discussed having a 'session database' that this sort of stuff (along with possibly the _full_ enviroment, pid, ....) into and then register accessing it via the new `ext` method on the databroker with a function that is `get_me_the_last_session_to_start_before_my_start_time(hdr)`

The session database is useful for other purposes and it is opt-in to get access to it (most users probably do not care).

**Comment by danielballan** [comment]
Closed by #1187
